pages/roadmap.md

```diff
@@ -1,72 +1,82 @@
 - **Send content with mentor emails:** [4428](https://github.com/exercism/exercism/issues/4428)
 
-- **Add ability to change which solutions are published:** Currently, you cannot change which solutions are published. We will enable this. [#4163](https://github.com/exercism/exercism/issues/4163)
+### Usability and documentation
 
-- **Enable commenting on community solutions:** We are going to let people decide whether want comments enable on a per solution basis.
+- **Make the steps of the installation guide linkable:** [#4470](https://github.com/exercism/exercism/issues/4470)
+- **Add more information about timescales for mentoring:** [#4368](https://github.com/exercism/exercism/issues/4368).
+- **Download link on all iterations** Add the download link and a link to instructions on all iterations in the student's view. [#4259](https://github.com/exercism/exercism/issues/4259)
 
-- **Reset CLI token:** We want to let people reset their CLI tokens in case they are leaked. [#4150](https://github.com/exercism/exercism/issues/4150)
+### Profiles and public solutions
 
-- **Download link on all iterations** Add the download link and a link to instructions on all iterations in the student's view. [#4259](https://github.com/exercism/exercism/issues/4259)
+- **Add ability to change which solutions are published:** Currently, you cannot change which solutions are published. We will enable this. [#4163](https://github.com/exercism/exercism/issues/4163)
+- **Allow users to change which iteraton is published:** At the moment only the most recent submisison is published. We will allow users to choose any of the iterations to make public.
+- **Allow deletion of iterations:** Often people submit iterations by default. This will allow people to delete them. [#3848](https://github.com/exercism/exercism/issues/3848)
+- **Add display name option:** [4365](https://github.com/exercism/exercism/issues/4365)
+- **Enable commenting on community solutions:** We are going to let people decide whether want comments enable on a per solution basis.
 
-- **Delete my account:** We need to give people the ability to delete their account without emailing Jeremy.
+### Maintainer Tools
+We want to update the new website to allow for an easy and pleasent experience for track maintainers and will build out tools within the UI to allow them to more easily experiment with track setups.
 
-- **Improve experience of legacy or indepedent solutions in mentored mode:** When importing from legacy or independent mode have sepererate restricted queues for core and side exercises.
+### Accessibility
 
-- **Allow deletion of iterations:** Often people submit iterations by default. This will allow people to delete them. [#3848](https://github.com/exercism/exercism/issues/3848)
+There are various a11y improvements we are making, spearheaded by [@unused's plan here](https://github.com/exercism/website/pull/182).
```

Comment:
I am not sure what `a11y` is supposed to mean here.

Reply:
Thanks. I've added a link to the https://a11yproject.com/